.github/workflows/release-suite-message-system-config.yml

```diff
@@ -0,0 +1,44 @@
+name: "[Release] suite message system config"
+
+permissions:
+  id-token: write # for fetching the OIDC token
+  contents: read # for actions/checkout
+
+on:
+  workflow_dispatch:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  release-config:
+    if: ${{ github.repository == 'trezor/trezor-suite' && (github.ref == 'refs/heads/release-message-system-production' || github.ref == 'refs/heads/release-message-system-develop') }}
```

Comment:
Why having a separate branch for develop? I dont really see the advantage in this.

Reply:
This is the reason: https://github.com/trezor/trezor-suite/pull/14233 
From time to time we need to test something that should never go to production and better than reverting it is to not merge it to develop at all.